### ISSUE TITLE
Uncaught TypeError: Cannot read property 'text' of undefined

### ISSUE THREAD

**Issue by svanharmelen** [redefiance/atom-go-find-references#2]
Didn't do anything specific to generate this error, but I keep getting it constantly...

**Atom Version**: 0.190.0
**System**: Mac OS X 10.9.5
**Thrown From**: [go-find-references](https://github.com/redefiance/atom-go-find-references) package, v1.1.4

### Stack Trace

Uncaught TypeError: Cannot read property 'text' of undefined

```
At /Users/svanharmelen/.atom/packages/go-find-references/node_modules/atom-tree-view/lib/tree-entry-view.coffee:13

TypeError: Cannot read property 'text' of undefined
  at TreeEntryView.module.exports.TreeEntryView.initialize (/Users/svanharmelen/.atom/packages/go-find-references/node_modules/atom-tree-view/lib/tree-entry-view.coffee:13:23)
  at TreeEntryView.View (/Users/svanharmelen/.atom/packages/go-find-references/node_modules/space-pen/lib/space-pen.js:184:25)
  at new TreeEntryView (/Users/svanharmelen/.atom/packages/go-find-references/node_modules/atom-tree-view/lib/tree-entry-view.coffee:6:3)
  at GoFindReferencesView.module.exports.GoFindReferencesView.showReference (/Users/svanharmelen/.atom/packages/go-find-references/lib/go-find-references-view.coffee:97:19)
  at /Users/svanharmelen/.atom/packages/go-find-references/lib/go-find-references-view.coffee:84:10
  at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:137:13)
  at emitOne (events.js:77:13)
  at Socket.emit (events.js:166:7)
  at readableAddChunk (_stream_readable.js:146:16)
  at Socket.Readable.push (_stream_readable.js:109:10)
  at Pipe.onread (net.js:517:20)

```

### Commands

```
     -7:48.3 editor:newline (atom-text-editor.editor)
     -7:47.4 snippets:next-tab-stop (atom-text-editor.editor)
     -7:47.4 snippets:expand (atom-text-editor.editor)
     -7:47.4 editor:indent (atom-text-editor.editor)
     -7:44.4 core:save (atom-text-editor.editor)
 15x -7:27.9 core:move-right (atom-text-editor.editor)
     -7:17.7 core:delete (atom-text-editor.editor)
     -7:16.3 core:backspace (atom-text-editor.editor)
     -7:15.3 core:save (atom-text-editor.editor)
     -7:08.6 core:backspace (atom-text-editor.editor)
     -7:08.0 core:save (atom-text-editor.editor)
     -7:04.8 core:backspace (atom-text-editor.editor)
     -7:04.4 core:save (atom-text-editor.editor)
     -7:01.8 core:backspace (atom-text-editor.editor)
     -7:01.6 core:save (atom-text-editor.editor)
  7x -6:27.6 core:backspace (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ],
    "disabledPackages": [
      "welcome",
      "deprecation-cop"
    ],
    "projectHome": "/Users/svanharmelen/Documents/GoCode/src",
    "destroyEmptyPanes": false
  },
  "go-find-references": {}
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v2.9.0
git-projects, v1.13.0
go-find-references, v1.1.4
go-plus, v3.3.6
go-rename, v0.3.0
monokai, v0.14.0
project-manager, v1.15.5
save-session, v0.12.5

# Dev
No dev packages
```

**Comment by redefiance** [comment]
thats what I get for using unversioned dependencies (:
should be fixed in 1.1.10

**Comment by svanharmelen** [comment]
@redefiance works like a charm again... Thanks for the quick fix and release!
